src/mapbox.common.ts

```diff
@@ -285,7 +295,9 @@ export interface MapboxApi {
   animateCamera(options: AnimateCameraOptions, nativeMap?: any): Promise<any>;
 
   setOnMapClickListener(listener: (data: LatLng) => void, nativeMap?): Promise<any>;
-
+  setOnPolygonClickListener(listener: (data: Polygon) => void, nativeMap?): Promise<any>;
```

Comment:
Of course, don't need these in common until iOS support lands. Or just mention safe-guarding the app code until it lands.